docs/documentation/server_admin/topics/authentication/passkeys.adoc

```diff
@@ -11,3 +11,60 @@ Therefore, users of {project_name} can do passkey registration and authenticatio
 Both synced passkeys and device-bound passkeys can be used for both Same-Device and Cross-Device Authentication (CDA).
 However, passkeys operations success depends on the user's environment. Make sure which operations can succeed in https://passkeys.dev/device-support/[the environment].
 ====
+
+[[_passkeys-conditional-ui]]
+==== Passkey Authentication with Conditional UI
+
+Passkey Authentication with Conditional UI can authenticate a user with its passkey in the same way as in xref:_webauthn_loginless[LoginLess WebAuthn].
+This authentication shows a user a list of passkeys stored on a device where the user runs a browser.
+Therefore, the user can select one of the passkeys in the list to authenticate them. Compared with xref:_webauthn_loginless[LoginLess WebAuthn], the authentication improves the user's experience of authentication.
+
+[NOTE]
+====
+This authentication uses https://github.com/w3c/webauthn/wiki/Explainer:-WebAuthn-Conditional-UI/[WebAuthn Conditional UI].
+Therefore, this authentication success depends on the user's environment.
+If the environment does not support WebAuthn Conditional UI, this authentication falls back to xref:_webauthn_loginless[LoginLess WebAuthn].
+====
+
+:tech_feature_name: Passkey Authentication
+:tech_feature_setting: -Dkeycloak.profile.feature.passkeys=enabled
+:tech_feature_id: passkeys
+include::../templates/techpreview.adoc[]
+
+.Procedure
+===== Setup
+
+Set up Passkey Authentication with Conditional UI as follows:
+
+. (if not already present) Register a new required action for WebAuthn passwordless support. Use the steps described in <<_webauthn-register, Enable WebAuthn Authenticator Registration>>. Register the `Webauthn Register Passwordless` action.
+
+. Configure the `WebAuthn Passwordless Policy`. Perform the configuration in the Admin Console, `Authentication` section,  in the tab `Policies` ->  `WebAuthn Passwordless Policy`. You have to set *User Verification Requirement* to *required* and *Require Resident Key* to *Yes* when you configure the policy for loginless scenario. Note that since there isn't a dedicated Loginless policy it won't be possible to mix authentication scenarios with user verification=no/resident key=no and loginless scenarios (user verification=yes/resident key=yes). Storage capacity is usually very limited on passkeys meaning that you won't be able to store many resident keys on your pass key.
```

Comment:
```suggestion
. Configure the `WebAuthn Passwordless Policy`. Perform the configuration in the Admin Console, `Authentication` section,  in the tab `Policies` ->  `WebAuthn Passwordless Policy`. Set *User Verification Requirement* to *required* and *Require Resident Key* to *Yes* when you configure the policy for loginless scenario. Note that since there is no dedicated Loginless policy, it is impossible to mix authentication scenarios with user verification=no/resident key=no and loginless scenarios (user verification=yes/resident key=yes). Storage capacity is usually very limited on passkeys meaning that you cannot store many resident keys on your pass key.
```

Reply:
Thank you. I will fix it as you suggested.